db/migrate/20200210074711_update_latest_blood_pressures_per_patient_per_months_to_version_2.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateLatestBloodPressuresPerPatientPerMonthsToVersion2 < ActiveRecord::Migration[5.1]
+  def change
+    drop_view :latest_blood_pressures_per_patient_per_quarters, materialized: true, revert_to_version: 1
+    drop_view :latest_blood_pressures_per_patients, materialized: true, revert_to_version: 1
+    update_view :latest_blood_pressures_per_patient_per_months,
+      version: 2,
+      revert_to_version: 1,
+      materialized: true
+    create_view :latest_blood_pressures_per_patient_per_quarters, materialized: true
+    create_view :latest_blood_pressures_per_patients, materialized: true
+  end
+end
```

Comment:
Can you describe briefly what schema changes are being made to the views?

Reply:
We added a new column in v2 called `responsible_facility` This is the `facility_id` of the last bp in the previous period, and is used to calculate missed visits